### PR TITLE
Tool 1126 swiftpm

### PR DESCRIPTION
**Pull request by lpusok**

### CODE REVIEW

**Comment by lpusok** [comment]
@trapacska pls add pullassigner

File: bitrise.yml [hunk 1]
```diff
@@ -3,36 +3,33 @@ default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
...
-        inputs:
-        - content: |-
-            #!/bin/bash
-            set -ev
-            git clone --single-branch -b "$SAMPLE_APP_BRANCH" "$SAMPLE_APP_URL" .
-    - certificate-and-profile-installer:
     after_run:
+    - test_objc_all
+
+  test_swiftpm:
```
Comment (BirmacherAkos) [inline bitrise.yml]: `test_swiftpm` WF is not called. 🤔

File: main.go [hunk 2]
```diff
@@ -52,6 +54,11 @@ func main() {
 	stepconf.Print(cfg)
```
Comment (BirmacherAkos) [inline main.go]: Why do we have this? 🤔 
cfg.OutputTool = "xcpretty"
https://github.com/bitrise-steplib/steps-xcode-build-for-test/pull/11/files#diff-7ddfb3e035b42cd70649cc33393fe32cR48

File: step.yml [hunk 3]
```diff
@@ -78,6 +78,17 @@ inputs:
...
+    opts:
+      title: Enable caching of Swift Package Manager packages
+      description: |-
+        Available options:
+        - `none` : Disable caching
+        - `swift_packages` : Cache Swift PM packages added to the Xcode project
+      value_options:
+      - "none"
+      - "swift_packages"
+      is_required: "true"
```
Comment (BirmacherAkos) [inline step.yml]: The `is_required` doesn't need to be string just the value_options of the input.

File: main.go (hunk 2 above)
Comment (lpusok) [reply main.go]: No idea.

File: bitrise.yml (hunk 1 above)
Comment (lpusok) [reply bitrise.yml]: Had an issue running the workflow due to codesigning issues.

**Pull request merged**